spec.html

```diff
@@ -39695,7 +39697,7 @@ <h1>Promise Abstract Operations</h1>
         <h1>PromiseCapability Records</h1>
         <p>A PromiseCapability is a Record value used to encapsulate a promise object along with the functions that are capable of resolving or rejecting that promise object. PromiseCapability Records are produced by the NewPromiseCapability abstract operation.</p>
```

Comment:
Rather than the putting the definition in the table below, I'd suggest

```suggestion
        <p>A <dfn>PromiseCapability Record</dfn> is a Record value used to encapsulate a promise object along with the functions that are capable of resolving or rejecting that promise object. PromiseCapability Records are produced by the NewPromiseCapability abstract operation.</p>
```

The usual style is to put the definition in the paragraph which introduces the term. Compare e.g. [ImportEntry Record](https://tc39.es/ecma262/#importentry-record), [Agent Events Record](https://tc39.es/ecma262/#sec-agent-event-records), etc.

Reply:
thanks, that's much cleaner